src/index.html

```diff
@@ -17,16 +17,34 @@
     />
   </head>
   <body>
-    <input
-      type="text"
-      data-qa="keypress"
-      placeholder="Try “Los Angeles“"
-    />
+    <form
+      class="search-form search-form__big"
+      data-qa="big"
+    >
+      <label class="search-form__label search-form__label__big">
+        <!-- prettier-ignore -->
```

Comment:
why ignore prettier?

Reply:
It is because when I made "npm run lint", the placeholder change from "Try &quot;Los Angeles&quot;" into 'Try "Los Angeles" ' and i could not came through it. So I found a solution on internet and it works